app/code/Magento/ProductAlert/Controller/Unsubscribe/Stock.php

```diff
@@ -12,6 +12,7 @@
 use Magento\Catalog\Api\ProductRepositoryInterface;
 use Magento\Framework\Controller\ResultFactory;
 use Magento\Framework\Exception\NoSuchEntityException;
```

Comment:
It looks like you have missed to add 
```
use Magento\Store\Model\StoreManagerInterface;
use Magento\Framework\App\ObjectManager;
```

Reply:
thanks for pointing it. :)

Reply:
yeah i added it in next commit now, thanks